tests/e2e/dsc_deletion_test.go

```diff
@@ -78,37 +78,42 @@ func (tc *testContext) testApplicationDeletion(component components.ComponentInt
 func (tc *testContext) testAllApplicationDeletion() error {
 	// Deletion all listed components' deployments
 
-	if err := tc.testApplicationDeletion(&(tc.testDsc.Spec.Components.Dashboard)); err != nil {
+	var err error
```

Comment:
Here as well, it has nothing to do with adding the component.

Reply:
Same answer, go linter version changed and it was throwing errors here. So, I had to clean up the code syntax.